### ISSUE TITLE
train子函数

### ISSUE THREAD

**Issue by yida88** [KaiyangZhou/deep-person-reid#159]
您好，在您的\torchreid\engine\engine.py中的def train(self)子函数中为什么只有 raise NotImplementedError 。他的训练过程代码定义在哪呢？我想学习下，非常感谢

**Comment by KaiyangZhou** [comment]
e.g. softmax for image reid: https://github.com/KaiyangZhou/deep-person-reid/blob/master/torchreid/engine/image/softmax.py
